pkgs/development/libraries/qt-5/qtModule.nix

```diff
@@ -74,4 +79,7 @@ mkDerivation (args // {
     maintainers = with maintainers; [ qknight ttuegel periklis bkchr ];
     platforms = platforms.unix;
   } // (args.meta or {});
+
+} // lib.optionalAttrs (stdenv.hostPlatform != stdenv.buildPlatform) {
+  dontWorryAboutQtMismatch = true;
```

Comment:
(Now, since I've been made aware of this PR... let's look into it...)

Is this an appropriate change? Mismatches in Qt will lead to severe issues down the line.

I suspect it's a cross-compilation idiosyncrasy here, but without a comment explaining it, I have no way to know for sure.

If a platform name ended-up breaking this check, the platform name should be added to the check.